examples/full_tutorial.ipynb

```diff
@@ -0,0 +1,882 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "d8ec9a76-0819-463d-8955-55789ee36400",
+   "metadata": {},
+   "source": [
+    "# Ipyreact walkthrough"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "083f2dc7-c0ec-482f-a720-aa2896ad193e",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "Welcome to this ipyreact walkthrough!   \n",
+    "The tutorial will be based on a very simple react button to show all the ipyreact features.  \n",
+    "\n",
+    "**Content** \n",
+    "* use the %react cell magic\n",
+    "* write a widget\n",
+    "* load CSS for your widget\n",
+    "* add parameters to your widgets (traitlets)\n",
+    "* interact with these parameters\n",
+    "* simple traitlet oberservation using `change`\n",
+    "* observe a traitlet and call python function\n",
+    "* observe a traitlet and call JavaScript function\n",
+    "* SOON: call python functions from JavaScript\n",
+    "* loading components from external files\n",
+    "* enable hot reloading  \n",
+    "* enable autocompletion in IDEs\n",
+    "* print a message at class initialization"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "06cb067d-0aea-4851-827d-d07f22466693",
+   "metadata": {},
+   "source": [
+    "First, we will use the **`%react` magic**  from ipyreact.  \n",
+    "The following line registers the cellmagic:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "de17b188",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#%pip install ipyreact\n",
+    "# This line is for JupyterLite (in case that this takes long, restart the kernel and run this cell again)"
```

Comment:
try
```
"# This line is for JupyterLite (sometimes this operation takes a long time and hangs, restart the kernel and run this cell again)"
```
or
```
"# This line is for JupyterLite (if this takes more than 10? seconds, something probably hung, restart the kernel and run this cell again)"
```
or

```
"# This line is for JupyterLite (this can hang, restart the kernel and try again after 15 seconds without a response)"
```
All of those are more direct phrases that read less awkwardly.

Reply:
Thanks!
I will also soon text the https://github.com/jupyterlite/xeus-python-demo kernel, then we won't need that line at all.